how-to/business-logic/how-to-change-user-role.md

```diff
@@ -0,0 +1,82 @@
+---
+title: How to Change User Role Programatically
+description: An example on changing the user role programatically, using cloud code
+type: how-to
+page_title: How to Change User Role Programatically
+slug: bl-change-user-role
+position:
+tags: backendservices, business-logic, cloud-code, user-role, cloud-functions
+teampulseid: 
+ticketid: 1016776
+pitsid: 
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Business Logic</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Cloud Functions</td>
+  </tr>
+</table>
+
+## Description
+
+You need to change/set a user's role programatically based on a certain logic (without going to {{site.tp}} portal).
+
+## Solution
+
+You can set up a Cloud Function to update the user role. Updating a user role requires the use of a [Master Key]({http://docs.telerik.com/platform/backend-services/javascript/security/security-master-key-auth}) - it is strongly recommended not to include your Master Key inside your app code - therefore a perfect solution is to use clode code as the code stays on {{site.tp}} server. 

```

Comment:
Let's make the solution more abstract - you can add such logic to the business logic layer.. w/o suggesting the cloud function directly,. After that let's mention a few examples how this can be used - expose a cloud function available to administrattors only or after a user account is created - check for certain conditions and update the role